### DISCUSSION TITLE
🌖 When Worlds Collide 🌖

### DISCUSSION THREAD

**Discussion by LivingWithHippos** [LivingWithHippos/unchained-android#274] [Feedback]
App is almost out of beta as all the basic features are implemented -> feedback is really appreciated, especially since a lot of changes were made in this version

### 🏷️ New features:

- [x] added youtube links support
- [x] added updates check
- [x] added cached torrent support
- [x] added cache status to search results (magnets only)
- [x] added torrent files selection
- [x] added torrent files re-selection
- [x] added custom download manager (supports external SD)
- [x] added libraries screen (thanks @Tgo1014)
- [x] added spanish translation (thanks @edgarpatronperez)

### 🔎  Enhancements:

- [x] updated default hosts handling
- [x] added sorting search results by seeders
- [x] updated search plugins

### 🦟 Bug fixes:

- [x] fixed crash if no browsing/media app are found (mainly on android tv)

#### Notes:

- as usual there are two APKs, a release and a debug one (the bigger one with "debug" in the name), the debug version will allow me to fix bugs more easily (it uses [Countly](https://count.ly/) for telemetry) but if your app does not crash you don' t need to use it
- both releases on GitHub have all the search plugins pre-installed
- it will be uploaded on Google Play a little later after getting feedback from the people using the GitHub version
- Fdroid version will be picked up automatically at some time

<hr /><em>This discussion was created from the release <a href='https://github.com/LivingWithHippos/unchained-android/releases/tag/v4.51.4-beta'>🌖 When Worlds Collide 🌖</a>.</em>

**Comment by Invictaz** [comment]
Great work. Will test later

**Reply by boognish-rising** [reply]
It would appear that I fell asleep last night at precisely the moment I had been going to post a reply here (I was able to react to @Invictaz comment but that was apparently the last thing I did before assing out) but I just wanted to say congrats on getting one step closer to going silver/RC!

I _was_ able to update Unchained to reflect this most recent update before conking out but obviously hadn't gotten a chance to take it for a test drive, as it were. I'll be sure to provide some feedback, even if only to say you nailed it.

**Reply by LivingWithHippos** [reply]
Thanks! There are around 1800 users and I usually test it on my phone and an emulator with an old android version but of course it's not enough to catch all the bugs.
